app.yml

```diff
@@ -66,7 +67,7 @@ default_permissions:
 
   # Deployments and deployment statuses.
   # https://developer.github.com/v3/apps/permissions/#permission-on-deployments
-  # deployments: write
+  deployments: write
```

Comment:
you need to change these permissions to `read` as well

Reply:
oops, my bad! done :)